### PR TITLE
fix(Testimonial): aria-label on Anchor 

### PR DESCRIPTION
**Pull request by zippyzow**
### Overview
New axe error emerged from `Testimonial` component: `Links must have discernible text`. This PR fixes this error by adding an aria-label to the `Anchor` in `Testimonial`. 

https://dequeuniversity.com/rules/axe/4.2/link-name?application=axeAPI

Screen Shot 2021-09-15 at 4 06 25 PM

 🏁 [Portal PR with this gamut fix and passing e2e test](https://github.com/codecademy-engineering/portal-app/pull/607)

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.79-alpha.f9d263.0<br/>@codecademy/gamut-labs@19.26.8-alpha.f9d263.0<br/>@codecademy/styleguide@44.6.1-alpha.f9d263.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61424d75c00cf2367fda23a0--gamut-preview.netlify.app>https://61424d75c00cf2367fda23a0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61424d75c00cf2367fda23a0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.79-alpha.f57a5d.0<br/>@codecademy/gamut-labs@20.0.0-alpha.f57a5d.0<br/>@codecademy/styleguide@44.6.1-alpha.f57a5d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://614250fd21ac3700b2cff6cb--gamut-preview.netlify.app>https://614250fd21ac3700b2cff6cb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/614250fd21ac3700b2cff6cb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.80-alpha.f5d190.0<br/>@codecademy/gamut-labs@20.0.1-alpha.f5d190.0<br/>@codecademy/styleguide@44.6.2-alpha.f5d190.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://614255639101340094541c86--gamut-preview.netlify.app>https://614255639101340094541c86--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/614255639101340094541c86>Deploy Logs</a></p>

File: packages/gamut-labs/src/Testimonial/index.tsx [hunk 1]
```diff
@@ -159,6 +159,7 @@ export const Testimonial: React.FC<TestimonialProps> = ({
       href={href}
       variant="interface"
       onClick={onClick}
+      aria-label="link"
```
Comment (borisonr) [inline packages/gamut-labs/src/Testimonial/index.tsx:162]: i'm not sure we want a string "link" to be read by screen readers? maybe a question for the accessibility channel?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.81-alpha.e515f6.0<br/>@codecademy/gamut-labs@20.0.2-alpha.e515f6.0<br/>@codecademy/styleguide@44.6.3-alpha.e515f6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://614281da3694202d57de9994--gamut-preview.netlify.app>https://614281da3694202d57de9994--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/614281da3694202d57de9994>Deploy Logs</a></p>

File: packages/gamut-labs/src/Testimonial/index.tsx (hunk 1 above)
Comment (zippyzow) [reply packages/gamut-labs/src/Testimonial/index.tsx]: Fixed

**Review by vonbarown — APPROVED** [review]
🏷️

**Review by JoshuaKGoldberg — APPROVED** [review]

**Review by katiezutter — APPROVED** [review]
🎉

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.81-alpha.5b8d84.0<br/>@codecademy/gamut-labs@20.0.2-alpha.5b8d84.0<br/>@codecademy/styleguide@44.6.4-alpha.5b8d84.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6143771a9101348ec5541d80--gamut-preview.netlify.app>https://6143771a9101348ec5541d80--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6143771a9101348ec5541d80>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.82-alpha.44fbdc.0<br/>@codecademy/gamut-labs@20.0.3-alpha.44fbdc.0<br/>@codecademy/styleguide@44.6.5-alpha.44fbdc.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61437ca277fd6600d50d68bc--gamut-preview.netlify.app>https://61437ca277fd6600d50d68bc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61437ca277fd6600d50d68bc>Deploy Logs</a></p>

**Pull request merged**